src/components/card/card.css

```diff
@@ -346,3 +348,53 @@ a.card-mode:focus .card-inner {
   -webkit-line-clamp: var(--max-lines);
   --max-lines: 9;
 }
+
+.premium-dogear {
+  --dogear-size: 61px;
+  --dogear-padding: 7px;
+  --dogear-icon: 18px;
+  --icon-size: var(--dogear-icon);
+  position: absolute;
+  padding: var(--dogear-padding) 0 0 var(--dogear-padding);
+  line-height: 0;
+  z-index: 1;
+  top: 0;
+  left: 0;
+}
+
+.premium-dogear::after {
+  content: " ";
+  background: var(--color--eb);
+  position: absolute;
+  display: block;
+  width: var(--dogear-size);
+  height: var(--dogear-size);
+  top: -1px;
+  left: -1px;
+  z-index: -1;
+  clip-path: polygon(0 0, 0% 100%, 100% 0);
+}
+
+.card--small-media--reverse .premium-dogear {
+  left: auto;
+  right: 0;
+  padding: var(--dogear-padding) var(--dogear-padding) 0 0;
+}
+
+.card--small-media--reverse .premium-dogear::after {
+  right: -1px;
+  left: auto;
+  clip-path: polygon(100% 0, 0 0, 100% 100%);
+}
+
+.premium-dogear ~ .article-meta {
+  padding-left: 40px;
+}
+
+.premium-dogear + .card-content .save-list-item {
+  padding-left: 25px;
+}
+
+.premium-dogear + .card-content-wrapper .card-meta {
+  padding-left: 15px;
+}
```

Comment:
Hvorfor disse paddings? Synes ikke rigtigt jeg kan se dem aktive i designsystemet via din branch.

Reply:
Jeg ser dem på test036

Reply:
Er der dog taget højde for plus artikler uden billede i horizontalScroll lister?
<img width="257" alt="Screenshot 2022-01-27 at 16 23 52" src="https://user-images.githubusercontent.com/7330250/151389028-a7cec913-b1f7-44f0-898d-324340a71781.png">